students/mattcasari/lesson03/mailroom.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+""" Lesson 3, Excercise 4
+
+@author: Matt Casari
+
+Link: https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/mailroom-part1.html
+
+Description:
+    The Program: Part 1
+    Write a small command-line script called mailroom.py. This script should be
+    executable. The script should accomplish the following goals:
+
+    It should have a data structure that holds a list of your donors and a 
+    history of the amounts they have donated. This structure should be 
+    populated at first with at least five donors, with between 1 and 3 
+    donations each. You can store that data structure in the global namespace.
+    The script should prompt the user (you) to choose from a menu of 3 
+    actions: “Send a Thank You”, “Create a Report” or “quit”.    
+
+"""
+
+# data list structure is [Name, [Amount1, Amount 2,...], Sum, Len, Average]
+data = [ ["Neil Armstrong", [15000.00, 15000.00],[],[],[]],
+["Buzz Aldrin", [23021.10, 25020.30, 28999.29],[],[],[]],
+["Sally Ride", [42917.42, 38281.28],[],[],[]],
+["Al Shepard", [2387.00,  2321.42, 3700.00],[],[],[]],
+["Alan Bean", [28477.13, 727.1],[],[],[]],
+["Chris Hadfield", [17325.42, 13823.83, 0.99],[],[],[]]]
+
+def prompt_user():
+    """ Prompts the user for menu option """
+    PROMPT_TEXT = ("\nSelect an option:\n"
+    "1. Send a Thank You\n"
+    "2. Create a Report\n"
+    "3. Quit\n"
+    "> ")
+
+    result = input(PROMPT_TEXT)
+    if result.isnumeric():
+        result = int(result)
+        if 0 < result < 4:
+            return result
+    else:
+        return False
```

Comment:
best to avoid functions with mixed data type returned - here you're either returning bool or int